documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -1094,3 +1095,28 @@ https://jdk.java.net/jmc/[JDK Mission Control].
 WARNING: Flight Recorder support is currently an _experimental_ feature. You're invited to
 give it a try and provide feedback to the JUnit team so they can improve and eventually
 <<api-evolution, promote>> this feature.
+
+[[stacktrace-pruning]]
+=== Stack trace pruning
+
+Since version 1.10, the JUnit Platform provides opt-out support for pruning stack traces
+produced by failing tests. This feature can be enabled or disabled via the
+`junit.platform.stacktrace.pruning.enabled` _configuration parameter_.
```

Comment:
```suggestion
Since version 1.10, the JUnit Platform provides built-in support for pruning stack traces
produced by failing tests. This feature can be disabled via the
`junit.platform.stacktrace.pruning.enabled` _configuration parameter_.
```